apps/GNOME/install

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+install_packages gnome gnome-core gnome-software gnome-shell || error "Could not install GNOME and its software."
```

Comment:
install_packages already prints to the terminal what it failed to install
you don't need to write any more output, just use exit 1